scripts/tokenomics-emulation/config.json

```diff
@@ -58,6 +59,12 @@
         "name": "ddc-subscribe",
         "amount": 1,
         "tier_id": 10
+      },
+      {
+        "name": "cere-app-to-user",
```

Comment:
We need to add `cere-user-to-app` here. And add support in the code.

Reply:
Added support